Livecheckables/wiredtiger.rb

```diff
@@ -1,4 +1,4 @@
 class Wiredtiger
-  livecheck :url   => "https://github.com/wiredtiger/wiredtiger/releases",
-            :regex => %r{href="/wiredtiger/wiredtiger/tree/([0-9\.]+)}
+  livecheck :url   => "https://github.com/wiredtiger/wiredtiger.git",
+            :regex => /^mongodb-v?(\d+(?:\.\d+)+)$/
```

Comment:
You misunderstood me. We're trying to _avoid_ the tags like `mongodb-4.5.0`, not _match_ them.

As I said in the previous review, just use the standard regex (without any modification): `/^v?(\d+(?:\.\d+)+)$/`

```suggestion
            :regex => /^v?(\d+(?:\.\d+)+)$/
```

Reply:
I see now, works for me.

Reply:
I see. 👍